### PR TITLE
 Added Keyboard shortcuts test case

### PR DESCRIPTION
**Pull request by furqanmlk**

### CODE REVIEW

**Review by stevemudie — APPROVED** [review]
Looks good @furqanmlk!

**Comment by stevemudie** [comment]
@furqanmlk I just had a thought about this though. 🤔 

This will affect the current keyboard shortcuts listing in-app. Is there a ticket and/or PR to make the needed change there as well?

File: data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+# (Required) Ensure all values are filled up
+name: "Allow CTRL/CMD + K to insert link formatting when text is selected"
```
Comment (lindalumitchell) [inline data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md:3]: Should be `V` instead of `K`?

Comment (lindalumitchell) [inline data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md:3]:
```suggestion
name: "Allow CTRL/CMD + V to insert link formatting when text is selected"
```

File: data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Login as **user-1**
+2. Copy **https://www.github.com**
+3. Type **Github**
```
Comment (lindalumitchell) [inline data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md:40]:
```suggestion
3. Type **Github** into message or reply input box
```

File: data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+4. Select a text and paste or press cmd/ctrl+v
```
Comment (lindalumitchell) [inline data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md:41]:
```suggestion
4. Select the text you typed, and paste or press CMD/CTRL+V
```

File: data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+5. Press enter
```
Comment (lindalumitchell) [inline data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md:42]:
```suggestion
5. Press Enter
```

File: data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+**Expected**
+
+3. Format should be as a markdown hyperlink with the selected text
+4. **Github** should be shown as a hyperlink
```
Comment (lindalumitchell) [inline data/test-cases/channels/keyboard-shortcuts/MM-Test-Case-1.md:47]:
```suggestion
4. **Github** should be shown as a hyperlink to https://github.com
```

**Review by lindalumitchell — CHANGES_REQUESTED** [review]
Just a few suggestions. :)

**Review by lindalumitchell — APPROVED** [review]
Thanks Malik! LGTM 👍

**Comment by saturninoabril** [comment]
Test case added as MM-T5421 - https://github.com/mattermost/mattermost-test-management/commit/6cf2eb454f7530265a634a9abbf248917de043f3#diff-47db6cd1a850f844f80a85761d88229e51d3136adcad1a1660795dbd4cbba246R28

**Pull request merged**